acceptance/localcluster/localcluster.go

```diff
@@ -415,17 +422,12 @@ func (l *Cluster) Start() {
 	l.createCACert()
 	l.createNodeCerts()
 	l.createClientCerts()
-
-	l.monitorStopper = make(chan struct{})
-	ch, err := l.client.MonitorEvents(nil, l.monitorStopper)
-	if err != nil {
-		panic(err)
-	}
-	go l.monitor(ch, l.monitorStopper)
-
 	for i := range l.Nodes {
 		l.Nodes[i] = l.startNode(i)
 	}
+
+	l.monitorStopper = make(chan struct{})
```

Comment:
what happens when the `startNode` operations fail? i'd leave this where it was above